hack/update-knative-components.go

```diff
@@ -0,0 +1,276 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"os"
+	"os/exec"
+	"time"
+
+	github "github.com/google/go-github/v68/github"
+)
+
+var (
+	fileScript = "hack/component-versions.sh"
+	fileJson   = "hack/component-versions.json"
+)
+
+// all the components that are kept up to date
+type Versions struct {
+	Serving  string
+	Eventing string
+	Contour  string
+}
+
+const versionsScriptTemplate = `#!/usr/bin/env bash
+
+# AUTOGENERATED FILE
+
+set_versions() {
```

Comment:
What about the KinD version?

Reply:
thanks for pointing it out! I will add it accordingly